.github/workflows/trigger-webui.yml

```diff
@@ -66,10 +58,33 @@ jobs:
           exit 1
 
       - name: Trigger anaconda run
+        id: trigger-anaconda-webui-automatic
+        continue-on-error: true
```

Comment:
I would rather go with the run always the new task. Seems to me more robust for extension.

Reply:
I think if I don't put continue-on-error and rather use the 'run: always' in the other task, the workflow will be marked as failed. But we dont want that, so have in CI red runs because of missing permissions.